File: apps/web/public/bridging/Day2/Java/ControlFlow.java
/**
 * Day 2 – Control flow (Java)
 * if/else and loops (for, while).
 * Compile: javac ControlFlow.java
 * Run:     java ControlFlow
 */
public class ControlFlow {

    public static void main(String[] args) {
        // --- if / else ---
        int score = 75;
        if (score >= 80) {
            System.out.println("Grade: A");
        } else if (score >= 60) {
            System.out.println("Grade: B");
        } else {
            System.out.println("Grade: C or below");
        }

        // --- for loop ---
        System.out.print("Count 1 to 5: ");
        for (int i = 1; i <= 5; i++) {
            System.out.print(i + " ");
        }
        System.out.println();

        // --- while loop ---
        int n = 3;
        while (n > 0) {
            System.out.println("Countdown: " + n);
            n--;
        }
    }
}

File: apps/web/public/bridging/Day2/C/control_flow.c
/*
 * Day 2 – Control flow (C)
 * if/else and loops (for, while).
 * Compile: gcc -o control_flow control_flow.c
 * Run:     ./control_flow
 */
#include <stdio.h>

int main(void) {
    /* --- if / else --- */
    int score = 75;
    if (score >= 80) {
        printf("Grade: A\n");
    } else if (score >= 60) {
        printf("Grade: B\n");
    } else {
        printf("Grade: C or below\n");
    }

    /* --- for loop --- */
    printf("Count 1 to 5: ");
    for (int i = 1; i <= 5; i++) {
        printf("%d ", i);
    }
    printf("\n");

    /* --- while loop --- */
    int n = 3;
    while (n > 0) {
        printf("Countdown: %d\n", n);
        n--;
    }

    return 0;
}
